SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -205,15 +207,39 @@ public class SwiftyTesseract {
     }
     
     // create unique file path
-    let filepath = try processPDF(images: images)
-    
+    let filepath = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString)
+    try processPDF(images: images, at: filepath)
+
     // get data from pdf and remove file
     let data = try Data(contentsOf: filepath)
```

Comment:
So when we call this, the tests are failing because the file on disk has actually changed and the filepath we've defined no longer exists. We need to append the `.pdf` path extension on anything that accesses the file after PDF processing has been performed